doc/website/blog/2022-06-announcing-cockroachdb-support-preview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Announcing Atlas CLI v0.4.2 with preview support for CockroachDB 🪳
+authors: Hedwigz
+tags: [cockroachdb, integration, announcement]
+image: https://blog.ariga.io/uploads/images/posts/cockroachdb/cockroachdb.png
+---
+
+Today, I'm happy to announce the release of [v0.4.2](https://github.com/ariga/atlas/releases/tag/v0.4.2) of the Atlas CLI. 
+This version includes many improvements and fixes, but I wanted to share with you  exciting news about something I
+personally worked on. As of v0.4.2, Atlas includes preview support for CockroachDB 🎉 
+
+## Atlas
+[Atlas](https://atlasgo.io) is an open-source project that helps developers better manage their database
+schemas. It has a [CLI tool](https://atlasgo.io/cli/reference) and a
+[Terraform integration](https://atlasgo.io/blog/2022/05/04/announcing-terraform-provider). By using Atlas's
+Data Definition Language (with a syntax similar to Terraform), users can plan, verify and apply changes
+to their databases in a simple, declarative workflow.
+Earlier this year, Atlas became the [migration engine for Ent](https://entgo.io/blog/2022/01/20/announcing-new-migration-engine),
+a widely popular, Linux Foundation backed entity framework for Go.
```

Comment:
```suggestion
a popular, Linux Foundation backed entity framework for Go.
```

Not sure if there is something like "widley" popular. Either it is popular id it is not.